src/sdk/account_sdk.js

```diff
@@ -73,11 +74,18 @@ class AccountSDK {
         }
     }
 
-    // copied from function in sts_sdk
     authorize_request_account(req) {
         const token = this.get_auth_token();
         // If the request is signed (authenticated)
         if (token) {
+            try {
+                signature_utils.check_access_key_status(token.access_key, this.requesting_account);
+            } catch (err) {
+                // GAP - until we change the IamError class to have templates
+                const message_with_details = `The security token included in the request is invalid`;
```

Comment:
there are no details in this message... why not have it in the error spec and just map it through our usual error code mappings?